src/boot/bootctl-status.c

```diff
@@ -503,6 +504,141 @@ int verb_status(int argc, char *argv[], void *userdata) {
         return r;
 }
 
+static int ref_file(Hashmap *known_files, const char *fn, const char *root)
+{
+        const char *p, *k = NULL;
+        int r, n;
+
+        assert(known_files);
+        if (!fn)
+                return -EINVAL;
+
+        p = prefix_roota(root, fn);
+
+        n = PTR_TO_INT(hashmap_get2(known_files, p, (void**)&k));
+        if (!k)
+                r = hashmap_put(known_files, strdup(p), INT_TO_PTR(1));
+        else
+                r = hashmap_update(known_files, p, INT_TO_PTR(++n));
+
```

Comment:
please remove this empty line. we usually try to keep function calls ant their immediate error handling close together. and use empty lines to spearate logic function calls.